src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -15,6 +15,26 @@ Imports Xunit
 
 Friend Module CompilationUtils
 
+    Public Function CreateCompilation(trees As IEnumerable(Of SyntaxTree),
+                                      Optional references As IEnumerable(Of MetadataReference) = Nothing,
+                                      Optional options As VisualBasicCompilationOptions = Nothing,
+                                      Optional assemblyName As String = Nothing) As VisualBasicCompilation
+        If options Is Nothing Then
+            options = TestOptions.ReleaseDll
+        End If
+
+        ' Using single-threaded build if debugger attached, to simplify debugging.
+        If Debugger.IsAttached Then
+            options = options.WithConcurrentBuild(False)
```

Comment:
It doesn't make them wrong either. I can follow up cleaning up both helpers. For now I'm gonna keep it in sync.